re/postgres/repository.go

```diff
@@ -53,12 +53,12 @@ const (
 		SET status = $2
 		WHERE id = $1
 		RETURNING id, name, domain_id, metadata, input_channel, input_topic, logic_type, logic_value,
-			output_channel, output_topic, recurring_time, recurring_type, recurring_period, created_at, created_by, updated_at, updated_by, status;  
+			output_channel, output_topic, start_datetime, recurring_time, recurring_type, recurring_period, created_at, created_by, updated_at, updated_by, status;  
 	`
 
 	listRulesQuery = `
 		SELECT id, name, domain_id, input_channel, input_topic, logic_type, logic_value, output_channel, 
-			output_topic, recurring_time, recurring_type, recurring_period, created_at, created_by, updated_at, updated_by, status
+			output_topic, start_datetime, recurring_time, recurring_type, recurring_period, created_at, created_by, updated_at, updated_by, status
```

Comment:
We'll probably need to improve and make this query smarter (so we eliminate some rules already here), but we'll do it in the follow-up PRs.